site/layouts/partials/footer.html

```diff
@@ -51,3 +51,23 @@
   crossorigin="anonymous"></script>
 <script src="{{ "js/tabs.js" | absURL }}"></script>
 <script src="{{ "js/main.js" | absURL }}"></script>
+{{ if (.Params.mermaid) }}
+<!-- MermaidJS support -->
+<script src="https://unpkg.com/mermaid@9.1.6/dist/mermaid.min.js"></script>
+<!-- MermaidJS initialization and configuration happen in the following block -->
+<script>
+    mermaid.initialize({
```

Comment:
Any reason we can't pre-render the diagrams and have them as images, rather than introduce this dependency?

Reply:
No technical reason.
Keeping them like this means no one needs an external app to update the drawing. Update the text and instantly see the changes. No rendering required. It also lowers the barrier to entry for rapidly adding diagrams whenever desired.

Reply:
Sure, how do I see the rendered output to fully gague the utility here?

Reply:
Using mermaid soounds fine to me